backend/src/test/scala/cromwell/backend/standard/callcaching/BlacklistCacheSpec.scala

```diff
@@ -1,22 +1,39 @@
 package cromwell.backend.standard.callcaching
 
 import cromwell.core.CacheConfig
-import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cromwell.services.CallCaching.CallCachingEntryId
+import org.scalatest.{FlatSpec, Matchers}
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
-class BlacklistCacheSpec extends FlatSpec with BeforeAndAfterAll with Matchers {
-  "The blacklist cache" should "default, blacklist and expire" in {
+class BlacklistCacheSpec extends FlatSpec with Matchers {
```

Comment:
Other test cases:
* The cache still returns `Unknown` for bucket B even if bucket A is in the cache
* If overfilled, the caches correctly eject (or reject?) entries

Reply:
considering how thin a layer this is over Guava caches that would be basically a Guava cache test...